Getting-Started.md

```diff
@@ -709,4 +853,58 @@ Kibiter then automatically generates a query as shown below.
 
 > :warning: **If you change the structure or delete it** you won't see the menu.
 
-> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
\ No newline at end of file
+> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
+
+#### Dump a mapping/data from an index [&uarr;](#dump-data/mapping-)
```

Comment:
`Dump the index mapping/data with elasticdump` since more appropiate since the section describes how to dump the mapping and whole data of an ES index.